csfieldguide/static/interactives/cmy-mixer/README.md

```diff
@@ -6,4 +6,5 @@ This interactive is used to create colours using CMY values.
 
```

Comment:
Can we please add some padding to this interactive? Maybe wrap it in a container if it's not already?

![image](https://user-images.githubusercontent.com/16066822/57894685-df1d1580-789b-11e9-8c5a-a99bcd06e1ec.png)

Reply:
same for RGB